src/main/java/org/gradlex/jvm/dependency/conflict/detection/rules/logging/Slf4jVersionCompatibilityRule.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright the GradleX team.
```

Comment:
Checkstyle currently requires that line, but I don't think this is how things work or should work, as:
* I'm not part of the Gradlex team
* you don't have any contributor agreement that I know of that would require copyright reassignment